test/token_helpers_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule TokenHelpersTest do
+  use ExUnit.Case, async: false
+  import Mock
+  doctest TokenHelpers
+
+  test "Gather User Input and Slugify" do
+    Mock.with_mock IO, [gets: fn(_prompt) -> "Jake\n" end] do
+      result = TokenHelpers._slug_input("This is a test string")
+      assert result == "Jake"
+    end
+  end
+
+  test "Input Checker returns token if not in array" do
+    array = [ "1", "2", "3", "4",
+              "5", "6", "7", "8",
+              "9", "X", "x"]
+
+    assert TokenHelpers._input_excluder(array, "a") == "a"
+  end
+
+  test "Input Checker loops until valid token is selected" do
+    array = [ "1", "2", "3", "4",
+              "5", "6", "7", "8",
+              "9", "X", "x"]
+
+    Mock.with_mock IO, [gets: fn(_prompt) -> "A\n" end] do
+      assert TokenHelpers._input_excluder(array, "7") == "A"
+    end
+  end
+
+  test "Create Token returns a token" do
+    Mock.with_mock IO, [gets: fn(_prompt) -> "A\n" end] do
+      assert TokenHelpers._create_token("this is a prompt") == "A"
+    end
+  end
```

Comment:
In general, I would say that if a function is called by something in another module, then it is a public function and should 1) not have an underscore at the beginning of its name, and 2) have unit tests.
I usually avoid writing tests for private functions (my mentor was against testing private functions) but sometimes I still write them when it seems helpful and when I don't want to create a whole new module for the private function. All this is to say that I'd probably remove the underscores from _create_token and _slug_input and then change the _input_excluder tests to actually be testing create_token or slug_input if it isn't too much trouble.

Reply:
_ scores have been removed from create token and slug input

Reply:
I will leave the tests since they have already been written